base/multidimensional.jl

```diff
@@ -1263,11 +1263,11 @@ end
 hash(x::Prehashed) = x.hash
 
 """
-    unique(itr[, dim])
+    unique(A::AbstractArray[, dim::Int])
 
-Returns an array containing only the unique elements of the iterable `itr`, in
-the order that the first of each set of equivalent elements originally appears.
-If `dim` is specified, returns unique regions of the array `itr` along `dim`.
+Return an array containing only the unique elements of array `A`, as determined by
```

Comment:
IMHO it doesn't hurt, can still be useful if you didn't read the signature carefully.